IFC4x3/Constants/u/USERDEFINED_30DlT5Cz5CEOn$eGTBzZaB.xml

```diff
@@ -0,0 +1,5 @@
+<?xml version="1.0" encoding="utf-8"?>
+<DocConstant xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" id="USERDEFINED_30DlT5Cz5CEOneGTBzZaB" Name="USERDEFINED" UniqueId="c036f745-33d1-4c39-8c7f-a1074bf6390b">
```

Comment:
You could re-use USERDEFINED and NOTDEFINED.

Reply:
Did that now.